.eslintrc.yml

```diff
@@ -593,8 +594,8 @@ overrides:
       '@typescript-eslint/no-use-before-define': off
       '@typescript-eslint/no-duplicate-imports': off # Superseded by `import/no-duplicates`
 
-      # Bellow rules are disabled because coflicts with Prettier, see:
-      # https://github.com/prettier/eslint-config-prettier/blob/master/%40typescript-eslint.js
+      # Below rules are disabled because they conflict with Prettier, see:
+      # https://github.com/prettier/eslint-config-prettier/blob/aeb4d52de54960ff48a8c627e6c25db9873d7719/index.js#L97-L110
```

Comment:
It's unrelated to this PR can you please extract it into a separate one?
Also please use non-versioned link https://github.com/prettier/eslint-config-prettier/blob/main/index.js